mailmerge/api.py

```diff
@@ -121,7 +151,7 @@ def addattachments(message, template_path):
         message.attach(part)
         print(">>> attached {}".format(normalized_path))
 
-    del message['attachments']
+    del message['attachment']
```

Comment:
Is this a bugfix?

Reply:
Yes, it is. It probably should go in a separate PR. Sorry about that! Will make that change.

Reply:
Removed in b99e752345d41e33a43558f121e29d6f69ab792e.